.github/workflows/test.yml

```diff
@@ -29,8 +25,6 @@ jobs:
       RAILS_ENV: test
     steps:
       - uses: actions/checkout@v4
-        with:
-          ref: ${{ inputs.github_sha || github.sha }}
```

Comment:
These two lines can be removed from the `jest` and `playwright` jobs further down in the workflow too